v2/inc/c_logging/log_context.h

```diff
@@ -110,6 +116,15 @@ uint32_t internal_log_context_get_values_data_length_or_zero(LOG_CONTEXT_HANDLE
     data_pos += sprintf(property_value_pair->value, __VA_ARGS__) + 1; \
     property_value_pair++; \
 
+#define EXPAND_SETUP_PROPERTY_PAIR_LOG_CONTEXT_WSTRING_PROPERTY(property_name, ...) \
+    /* Codes_SRS_LOG_CONTEXT_07_001: [ LOG_CONTEXT_WSTRING_PROPERTY shall expand to code allocating a property/value pair of type wchar_t_ptr and the name property_name. ]*/ \
+    property_value_pair->value = data_pos; \
+    property_value_pair->name = MU_TOSTRING(property_name); \
+    property_value_pair->type = &wchar_t_ptr##_log_context_property_type; \
```

Comment:
>wchar_t_ptr##_log_context_property_type; [](http://example.com/codeflow?start=33&length=40)

Mu_C2?

Reply:
for some reason which I don't understand, MU_C2 doesn't work here

Reply:
Try using MU_C2A... see what happens if it works. Totally wild guess.

Reply:
yeah, MU_C2A works, modified to use MU_C2A